src/posix/platform/radio.cpp

```diff
@@ -96,10 +96,11 @@ void otPlatRadioSetPromiscuous(otInstance *aInstance, bool aEnable)
 
 void platformRadioInit(otUrl *aRadioUrl)
 {
-    ot::Posix::RadioUrl &radioUrl       = *static_cast<ot::Posix::RadioUrl *>(aRadioUrl);
-    bool                 resetRadio     = (radioUrl.GetValue("no-reset") == nullptr);
-    bool                 restoreDataset = (radioUrl.GetValue("ncp-dataset") != nullptr);
-    const char *         urlFemLnaGain  = radioUrl.GetValue("fem-lnagain");
+    ot::Posix::RadioUrl &radioUrl        = *static_cast<ot::Posix::RadioUrl *>(aRadioUrl);
+    bool                 resetRadio      = (radioUrl.GetValue("no-reset") == nullptr);
+    bool                 restoreDataset  = (radioUrl.GetValue("ncp-dataset") != nullptr);
+    const char *         urlFemLnaGain   = radioUrl.GetValue("fem-lnagain");
+    const char *         urlCcaThreshold = radioUrl.GetValue("ccathreshold");
```

Comment:
```suggestion
    const char *         urlCcaThreshold = radioUrl.GetValue("cca-threshold");
```
nit: follow the same naming convention of other parameters to use `-` to separate words?

Reply:
But in CLI it's one word `ccathreshold`. Should we keep consistent about this?

Reply:
Just a weak recommendation that radio URL parameters keep consistent with each other.

Reply:
I have a slight preference to maintain consistency among URL parameters.

Reply:
Updated to `cca-threshold`.